backend/grupo-1-tendencias-todo-app-firebase-adminsdk-vl65h-7ea3fd91c4.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
+  "project_id": "grupo-1-tendencias-todo-app",
+  "private_key_id": "7ea3fd91c4677d8f0dcd079553efb82aa4b3edb4",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQCpFmGKlo10HJG9\nD245Z8mad4kA/2hJPx6ixdS3Cann4gqLwA85M4PTdWbfKcha5kmrEBKDiWG7n6kG\nYQdK6w8/DWCtSe4B1ak1bxI4qzX294LCjKL6vtViMFHm2aogiMPoNDhHnoL8x+2b\n0I+toOFMDGGpqq7F/CNtKPyc/9OFRGj73+VuktaXpcQVCCqC8Ke3Njwbbrd37hDW\nI4amV1TykQPfaOow87us9lrFuNnH0XEuLgRei51xTbQv0tYORFn636AK95Au3Lnj\nLekYkjNoJushWBcPDYHsuVAfZtJ9w5bu4ph9+T/1YMOGofIfxT4ApsWMZAa0XbM1\nOEVaytfrAgMBAAECggEAMH3drWwt1w3Wcfz55HayZ50+ak3VL6Rtecr4jSG1W24F\nDj0fYI04ki1KOlf2QLNojymF1blRkQhh5hVvjPnKno+MmUXpfxcydsOK2zumt1QW\naQsCNp0B/y3SXz0vQDNY08AeH9QCxtHjf9sZSXcpBbFLVOhBRUBhPI4TnEb193LF\nwWtfYjRur3iUIN4eHUJmqJxRGinOAapKfAFxZdssfyAflYZ9h9AoDvdiuEFK3z6V\nFRTntanTGE3lpYyZ5iH28lOZNkjjIbYComFvOaLRcWnXKcHvE/9oWkH8Xy0IGWUw\nemps8AiwHReZNqNGmzE6iBHzJzPAo1m6Hv8x/56fAQKBgQDspw0PSW0+M8AU94Ol\n91+Mdr0uuMN6yJ3vyN2VWFG7X/6cJ/yHoYKbhEmCS0XZAvZMvU/2/fb4xSFx5g8S\ni7+Vp0CRUnUJddHAiRORz8DfHUN46eyOCwlGNIbQjngw1lviw9fn/CglbsYAhR1+\ndFlC+hejhms7KHsXMUi/dAHkgQKBgQC26T75NdylBJOLHB//ZfQFS5Ly9HAz7xgs\naE0MDTh+1MX9j942QwjLEa1vgkfrTyPmPW+ecWiR9L5ou680POfGN0hvvf9lB+QO\nHj0+Dtx7xo9fJy7UiwgHVs4XiwdnVpAJRnmcdwHSmSHtyP+pLcKOoONYp1Y/nsIo\nnIX9iL1WawKBgQDgzDadpNQ+2A3IXO4TQ47h8RD4Rs553gQY+Kdz7DeHVEKZt2os\ntIFKlTj2qeYppKSudM53AvwNh8SjWzRKG7HbyMqOwNMoL2EWEaAjTEQcLMIhI6Rv\nsZAXvmGpzwvg6F7UY9C+L0s7c6Rmbgx8SK1Z8BNwZImL4oI6n7hJFTuPAQKBgQCO\nXem4C2Zbwr4cbp2xKmfOn04XcI7MvXKxTdV6ZeftQCuC+rHYxkBX3sCBpIPZ8VVd\n5b+1MUxVHov99Oef3y//phPpdZcNyiCFpJ/CZ8rRRHFffJ4ErNpo6/WpeXsfwc6G\niaYqVK7EfpBcBjCIO6fNYAjUfhqxlh5Fgd8AwaetVQKBgHr4VMSHgiqhnicjiL9u\nfhqmUXCmYS91hBrTDhzmUD3Wan8fhWQr5vRP5oyX2CgY0rpf9HK9N6tnt7TgI3VU\nXFpEmXpEGcW0ppECBqvWUcXQDUWuSYZotjz8ewk8ErW6c4XdSrm3fQPr6lWKv/pU\nUyAV41aOmCVO9FzvwZJOs0Gy\n-----END PRIVATE KEY-----\n",
```

Comment:
You shouldn't commit this file to the repo, as it exposes sensitive data. You could save this data as environment variables which are not publicly exposed.

Try to follow these instructions: https://firebase.google.com/docs/admin/setup#initialize-without-parameters